src/index.html

```diff
@@ -11,6 +11,39 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://airbnb.com.us"
+    method="post"
+    class = "form"
+    data-qa="big"
+    >
+      <input class = "element"
```

Comment:
Check this moment for other tags as well

```suggestion
      <input class="element"
```